Makefile.am

```diff
@@ -35,6 +35,10 @@ libcrun_SOURCES = src/libcrun/utils.c \
 			src/libcrun/signals.c \
 			src/libcrun/seccomp_notify.c
 
+libocispec/libocispec.la:
+	$(MAKE) $(AM_MAKEFLAGS) -C libocispec libocispec.la
+
+crun$(EXEEXT): libocispec/libocispec.la
```

Comment:
I don't think this works, because now "make crun" **only** builds libocispec/libocispec.la

So you don't need this part, because automake already generates rules for all programs:

`crun$(EXEEXT): $(crun_OBJECTS) $(crun_DEPENDENCIES) $(EXTRA_crun_DEPENDENCIES) `

Deleting the line makes it work again, the important part was the  libocispec/libocispec.la rule. Thanks!

Reply:
thanks for the review.  Pushed a fixed version